src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("Mate Java");
+        book.setPrice(BigDecimal.valueOf(39.99));
+
+        Book book2 = bookDao.create(book);
+        System.out.println(book2);
+
+        Optional<Book> bookOptional = bookDao.findById(book2.getId());
+        System.out.println(bookOptional.isPresent());
+        System.out.println(bookOptional.get().getTitle());
+        System.out.println(bookOptional.get().getPrice());
```

Comment:
Using `bookOptional.get()` without checking if the value is present can lead to a `NoSuchElementException`. Consider using `bookOptional.ifPresent()` or checking `isPresent()` before accessing the value.